Content.Shared/Disposal/Components/SharedDisposalUnitComponent.cs

```diff
@@ -21,6 +21,12 @@ public abstract class SharedDisposalUnitComponent : Component
         [DataField("mobsCanEnter")]
         public bool MobsCanEnter = true;
 
+        /// <summary>
+        /// Automatically flushes items and has no pressure requirement
+        /// </summary>
+        [DataField("autoFlush"), ViewVariables(VVAccess.ReadWrite)]
+        public bool AutoFlush = false;
```

Comment:
Okay, after testing i see this is supposed to be more of an instant flush that ignores pressure, so rename this and adjust the comment (also seems a little too powerful, make it at least require pressing flush)